docs/pages/access-controls/guides/role-templates.mdx

```diff
@@ -184,27 +208,27 @@ spec:
       '*': '*'
 ```
 
-A GitHub connector `github.yaml` maps every `cyber` team's member of organization `octocats` to
-the role `sso-users`:
+A GitHub connector called `github.yaml` maps every member of team `cyber` in
+organization `octocats` to the role `sso-users`:
 
 ```yaml
 kind: github
 version: v3
 metadata:
   name: github
 spec:
-  # Client ID of Github OAuth app
+  # Client ID of GitHub OAuth app
   client_id: client-id
-  # Client secret of Github OAuth app
+  # Client secret of GitHub OAuth app
   client_secret: secret-data-here
-  # Connector display name that will be shown on web UI login screen
-  display: Github
+  # Connector display name that will be shown on the Web UI login screen
+  display: GitHub
   # Callback URL that will be called after successful authentication
   redirect_url: https://teleport.example.com/v1/webapi/github/callback
   # Mapping of org/team memberships onto allowed logins and roles
   teams_to_logins:
```

Comment:
This is a confusing name that remains for historical reasons.

Would it be worth adding a comment that despite the `teams_to_logins` name, this is actually mapping GitHub teams to Teleport roles (not logins)?